src/main/java/org/miracum/streams/ume/obdstofhir/mapper/mii/TumorkonferenzMapper.java

```diff
@@ -0,0 +1,90 @@
+package org.miracum.streams.ume.obdstofhir.mapper.mii;
+
+import de.basisdatensatz.obds.v3.TumorkonferenzTyp;
+import io.micrometer.common.lang.NonNull;
+import java.util.ArrayList;
+import org.hl7.fhir.r4.model.CarePlan;
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.Identifier;
+import org.hl7.fhir.r4.model.Reference;
+import org.miracum.streams.ume.obdstofhir.FhirProperties;
+import org.miracum.streams.ume.obdstofhir.mapper.ObdsToFhirMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TumorkonferenzMapper extends ObdsToFhirMapper {
+  private static final Logger LOG = LoggerFactory.getLogger(TumorkonferenzMapper.class);
+
+  public TumorkonferenzMapper(FhirProperties fhirProperties) {
+    super(fhirProperties);
+  }
+
+  public CarePlan map(
+      @NonNull TumorkonferenzTyp tk, @NonNull Reference patient, Reference primaerDiagnose) {
+
+    CarePlan carePlan = new CarePlan();
+    // Meta
+    carePlan.getMeta().addProfile(fhirProperties.getProfiles().getMiiPrOnkoTumorkonferenz());
+    // Identifier + Id
+    final Identifier identifier =
+        new Identifier()
+            .setSystem(fhirProperties.getSystems().getTumorkonferenzId())
+            .setValue("Tumorkonferenz_" + tk.getTumorkonferenzID());
+    carePlan.addIdentifier(identifier);
+    carePlan.setId(computeResourceIdFromIdentifier(identifier));
+    // Status
+    if (tk.getMeldeanlass().equals(TumorkonferenzTyp.Meldeanlass.BEHANDLUNGSENDE)) {
+      carePlan.setStatus(CarePlan.CarePlanStatus.COMPLETED);
+    } else {
+      carePlan.setStatus(CarePlan.CarePlanStatus.ACTIVE);
+    }
+    // intent
+    carePlan.setIntent(CarePlan.CarePlanIntent.PROPOSAL);
```

Comment:
```suggestion
    carePlan.setIntent(CarePlan.CarePlanIntent.PLAN);
```

Hör lieber nicht auf mich, es ist doch im IG als "plan" hard-codiert... Ich hab nochmal nachgefragt ob Proposal doch sinnvoller ist: https://mii.zulipchat.com/#narrow/channel/255455-MII-Onko/topic/Therapieabweichung/near/503284376

🙈